### PR TITLE
enhancement/LAYOUT-Update-layout

### PR DESCRIPTION
**Pull request by zanusilker**
- Update layout

### CODE REVIEW

File: src/layouts/app/layout/side-menu.js [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
+import { Collapse, Divider, Drawer as MuiDrawer, IconButton, List, ListItemButton, ListItemIcon, ListItemText, ListSubheader, Stack, styled } from '@mui/material';
 
 // local dependencies
-import controller from './controller';
-import DrawerHeader from './drawer-header';
+import { controller } from '../controller';
 
 // constants
-import * as ROUTES from 'constants/routes';
+import { HEADER_HEIGHT } from '../../../hooks/use-free-height';
```
Comment (serejabogdan) [inline src/layouts/app/layout/side-menu.js:13]: use import like 'hooks/use-free-height'

File: src/layouts/app/layout/side-menu.js [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
     duration: theme.transitions.duration.leavingScreen
   }),
   overflowX: 'hidden',
   width: `calc(${theme.spacing(7)} + 1px)`,
-  [theme.breakpoints.up('sm')]: {
-    width: `calc(${theme.spacing(7)} + 1px)`
-  }
 });
 
 const openedMixin = (theme) => ({
```
Comment (serejabogdan) [inline src/layouts/app/layout/side-menu.js:26]: remove round brackets
(theme) -> theme

File: src/layouts/app/layout/side-menu.js [hunk 3]
```diff
@@ -39,7 +35,7 @@ const openedMixin = (theme) => ({
 const Drawer = styled(MuiDrawer, {
   shouldForwardProp: (prop) => prop !== 'open'
```
Comment (serejabogdan) [inline src/layouts/app/layout/side-menu.js:36]: remove round brackets

File: src/layouts/app/menu.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import * as ROUTES from 'constants/routes';
```
Comment (serejabogdan) [inline src/layouts/app/menu.ts:1]: // constants
import * as ROUTES from 'constants/routes';

File: src/layouts/app/menu.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+import { MENU_TYPE } from './layout/side-menu';
```
Comment (serejabogdan) [inline src/layouts/app/menu.ts:2]: // local dependencies

File: src/layouts/app/menu.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+import { AssignmentIndOutlined, FormatListBulletedOutlined, StarBorderOutlined, SupervisorAccountOutlined, VpnKeyOutlined } from '@mui/icons-material';
```
Comment (serejabogdan) [inline src/layouts/app/menu.ts:3]: // outsource dependencies

**Review by serejabogdan — CHANGES_REQUESTED** [review]

**Review by alexandr-feschenko — APPROVED** [review]

**Pull request merged**